lib/mock/mock-utils.js

```diff
@@ -230,25 +230,20 @@ function mockDispatch (opts, handler) {
   const key = buildKey(opts)
   const mockDispatch = getMockDispatch(this[kDispatches], key)
 
+  mockDispatch.timesInvoked++
```

Comment:
We now increase `timesInvoked` instead of decreasing `times` (which is now constant).

(Is `invoked` a good name, or would something like `timesIntercepted` be better?)